package.json

```diff
@@ -34,6 +35,7 @@
     "@types/react": "18.0.21",
     "@types/react-dom": "18.0.6",
     "@types/styled-components": "^5.1.26",
+    "@types/uuid": "^8.3.4",
```

Comment:
r we using both uuid and @types/uuid? if not maybe remove the one not being used

Reply:
`@types/uuid` is the type definition for `uuid` with typescript It is a dev dependency and isn't bundled. This is supposed to be the case! :)